packages/react/src/components/CardEditor/CardEditor.jsx

```diff
@@ -248,36 +360,88 @@ const CardEditor = ({
             testId={`${testID || testId}-card-gallery-list`}
           />
         ) : (
-          <CardEditForm
-            cardConfig={finalCardToEdit}
-            isSummaryDashboard={isSummaryDashboard}
-            onChange={onChange}
-            dataItems={dataItems}
-            getValidDataItems={getValidDataItems}
-            getValidTimeRanges={getValidTimeRanges}
-            onValidateCardJson={onValidateCardJson}
-            onCardJsonPreview={onCardJsonPreview}
-            availableDimensions={availableDimensions}
-            i18n={mergedI18n}
-            currentBreakpoint={currentBreakpoint}
-            dataSeriesItemLinks={dataSeriesItemLinks}
-            onFetchDynamicDemoHotspots={onFetchDynamicDemoHotspots}
-          />
+          <>
+            {showJsonEditor ? (
+              <CardCodeEditor
+                onSubmit={(card, setError) =>
+                  handleSubmit(
+                    card,
+                    cardConfig.id,
+                    setError,
+                    onValidateCardJson,
+                    onChange,
+                    setShowJsonEditor
+                  )
+                }
+                onClose={() => setShowJsonEditor(false)}
+                initialValue={modalJsonData}
+                i18n={pick(
+                  mergedI18n,
+                  'errorTitle',
+                  'modalTitle',
+                  'modalLabel',
+                  'modalHelpText',
+                  'modalIconDescription',
+                  'copyBtnDescription',
+                  'copyBtnFeedBack',
+                  'expandBtnLabel',
+                  'modalPrimaryButtonLabel',
+                  'modalSecondaryButtonLabel'
+                )}
+              />
+            ) : null}
+            <CardEditForm
+              cardConfig={finalCardToEdit}
+              isSummaryDashboard={isSummaryDashboard}
+              onChange={onChange}
+              dataItems={dataItems}
+              getValidDataItems={getValidDataItems}
+              getValidTimeRanges={getValidTimeRanges}
+              onValidateCardJson={onValidateCardJson}
+              onCardJsonPreview={onCardJsonPreview}
```

Comment:
This has been removed from CardEditForm correct ? please remove unwanted props from passing into CardEditForm

Reply:
Updated